core/server/master/src/main/java/alluxio/master/file/FileSystemMasterAuditContext.java

```diff
@@ -125,6 +127,17 @@ public FileSystemMasterAuditContext setSrcInode(Inode srcInode) {
     return this;
   }
 
+  /**
+   * Sets mCreateTimeMillis field.
+   *
+   * @param createTimeMillis the start time of this operation
+   * @return this {@link AuditContext} instance
+   */
+  public FileSystemMasterAuditContext setCreateTimeMillis(Long createTimeMillis) {
```

Comment:
Long -> long

Reply:
Agree, Unnecessary Boxing may introduce overhead

Reply:
it will be modified